schema/google/showcase/v1beta1/echo.proto

```diff
@@ -361,9 +361,7 @@ message PagedExpandResponseList {
 
 message PagedExpandLegacyMappedResponse {
   // The words that were expanded, indexed by their initial character.
-  // (-- aip.dev/not-precedent: This is a legacy, non-standard pattern that violates
```

Comment:
I don't think we want to remove this. GCE uses this type of "mapped-paged" response in several RPCs, which is why this RPC was put in Showcase. How does PHP handle building the GCE GAPIC?

Reply:
That's interesting, I don't think we handle `map` for pagination in GCE. That may be a bug on our end.

Would you be able to point to an example in the protos (or in a doc) where this exists and how we should be handling it?